sharding/config/config.go

```diff
@@ -21,7 +21,7 @@ type MultiClusterConfig struct {
 	//Cluster name
 	Cluster string `yaml:"Cluster"`
 	//Cluster weight
-	Weight int `yaml:"Weight"`
+	Weight float64 `yaml:"Weight"`
```

Comment:
We already have validator for that. And putting code above does not work "out of the box".